tests/meta-mender-ci/recipes-testing/mender-coverage-client/mender-client_%.bbappend

```diff
@@ -0,0 +1,67 @@
+SUMMARY = "Build a Mender client which records coverage during its invocation"
+
+DEPENDS += "gobinarycoverage rsync-native"
+
+do_copy_src_to_build[nostamp] = "1"
```

Comment:
Why do you need `nostamp`?

Reply:
To make sure it is run everytime the recipe is executed